Bottom: 6c2a68f7c4847f75cca18d0ec8e78fbf46a3177a
Top:    8d3b6ff23eae2fcf2929cc9066f58169582e56f7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-05-16 20:50:23 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 1c755f7..af7ef91 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2459,7 +2459,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
